From 0ce981ad2719d2532120f82a37ba0a12827261f0 Mon Sep 17 00:00:00 2001
From: Valentin David <me@valentindavid.com>
Date: Sun, 26 Oct 2025 16:06:02 +0100
Subject: [PATCH 5/7] openssh: Include ssh*_config.d/*.conf

Most distributions include system-wide configuration fragments from
and some software expect it: systemd, libvirt.
---
 elements/components/openssh.bst | 14 ++++++++++++++
 1 file changed, 14 insertions(+)

diff --git a/elements/components/openssh.bst b/elements/components/openssh.bst
index 04758d472..948640fbf 100644
--- a/elements/components/openssh.bst
+++ b/elements/components/openssh.bst
@@ -35,6 +35,20 @@ variables:
     %{arch_options}
 
 config:
+  configure-commands:
+    (<):
+    - |
+      cat <<EOF >>"sshd_config"
+
+      Include %{sysconfdir}/sshd_config.d/*.conf
+      EOF
+
+    - |
+      cat <<EOF >>"ssh_config"
+
+      Include %{sysconfdir}/ssh_config.d/*.conf
+      EOF
+
   install-commands:
     (>):
     - |
-- 
2.52.0

